internal/dynamicanalysis/analysis.go

```diff
@@ -28,6 +32,24 @@ var resultError = &Result{
 	},
 }
 
+func ParseURLsFromSSlStripOutput(content []byte) []string {
```

Comment:
It would be more efficient if this method took an `io.Writer` (pass in the opened file), rather than reading all the bytes in.

Reply:
you still have to parse it so you ought to read the file.
Or would you rather I'll read the file inside the function?